composer.json

```diff
@@ -16,6 +16,7 @@
 
     "require": {
         "php": ">=5.4.0 <8.0",
+        "aws/aws-sdk-php": "^3.8",
```

Comment:
This is a very big dependency for a very small, very specific functionality. Also it raises required version of PHP to 5.5.
Please move this dependency to `suggest` section and document it similarly to jsonpath dependency in http://codeception.com/docs/modules/REST#seeResponseJsonMatchesJsonPath

Reply:
Done moved to suggested section